example/lib/screens/home.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:airship_example/styles.dart';
+import 'package:airship_example/bloc/airship_bloc.dart';
+
+class _HomeState extends State<Home> {
+  final AirshipBloc _airshipBloc = AirshipBloc();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
is it possible to break this into smaller pieces and combine them? I find it pretty hard to follow with all the nesting.

Reply:
removed the button into its own widget to make this a little simpler